.github/workflows/merge-gatekeeper.yml

```diff
@@ -0,0 +1,26 @@
+name: Merge Gatekeeper
```

Comment:
this file seems new, can you provide a bit of context on why this is needed. Thank you @jitu5

Reply:
@ravi-kumar-pilla 
The merge-gatekeeper workflow is triggered by pull requests on the main branch. It checks every 30 seconds to see if other specified workflows have completed and passed. If any of these workflows fail, the merge-gatekeeper will also fail. This workflow must pass for pull requests to be mergeable. Also, it is configured to ignore certain CircleCI workflows as specified in the ignored field. 
once this PR is merge will need to mark this workflow as required.
<img width="1058" alt="Screenshot 2024-04-18 at 10 18 49 a m" src="https://github.com/kedro-org/kedro-viz/assets/38945204/0db03a82-7ec7-40d6-9ff4-2f43bf0dda76">